client/app/projects/blog/blog.controller.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+angular.module('observatory3App')
+.controller('ProjectsBlogCtrl', function ($scope, $http, $stateParams) {
+    $scope.load = function() {
```

Comment:
Okay cool. That makes sense. Just wasn't sure and didn't look at it long enough to realize that.